sources/api/apiserver/src/server/mod.rs

```diff
@@ -277,6 +319,37 @@ async fn get_settings(
     Ok(SettingsResponse(settings))
 }
 
+/// Prints a more specific message before exiting through usage().
+fn usage_msg<S: AsRef<str>>(msg: S) -> ! {
+    eprintln!("{}\n", msg.as_ref());
+    process::exit(2);
+}
+
+/// Constructs a Key Pair Map from the
```

Comment:
This comment seems incomplete.

Reply:
Also, do we even need this comment ? The function name is descriptive enough IMO.